support/ci/should_we_build.sh

```diff
@@ -14,11 +14,7 @@ echo "======================================================="
 echo "Testing whether or not we need to build ${package_name}"
 echo "======================================================="
 
-cat .bldr.toml | \
-    rq --input-toml | \
-    jq --exit-status '.["'${package_name}'"]' &>/dev/null
-
-if [ $? != 0 ]; then
+if ! rq -t < .bldr.toml | jq -e ".[\"${package_name}\"]" &>/dev/null; then
```

Comment:
See https://github.com/koalaman/shellcheck/wiki/SC2181#check-exit-code-directly-with-eg-if-mycmd-not-indirectly-with-
and https://github.com/koalaman/shellcheck/wiki/SC2002

I also shortened `--input-toml` and `--exit-status` to `-t` and `-e` to fit more nicely on one line, but there shouldn't be any functional change.